src/seedlink/plugins/serial_plugin/CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 SET(GCF_DIR ${CMAKE_CURRENT_SOURCE_DIR}/../libgcf2-2.3.2)
 SET(GCF_INC ${GCF_DIR}/src/gcf2.h)
 
+CHECK_FUNCTION_EXISTS(cfmakeraw HAVE_CFMAKERAW)
+CONFIGURE_FILE(${CMAKE_CURRENT_SOURCE_DIR}/serial_config.h.in ${CMAKE_CURRENT_SOURCE_DIR}/serial_config.h)
```

Comment:
Instead of configuring a file you could also pass that as define during compile time:
```cmake
IF(HAVE_CFMAKERAW)
        MESSAGE(STATUS "Found function cfmakeraw")
        ADD_DEFINITIONS(-DHAVE_CFMAKERAW)
ENDIF(HAVE_CFMAKERAW)
```
I personally prefer this way. Having an additional .in file reminds me on the awful days with autotools. ;) Apart from my personal opinion we do not pollute the source directory which is the concept of cmake. All build files must go into the build directory.